pontoon/machinery/gpt/openai_connection.py

```diff
@@ -0,0 +1,68 @@
+from openai import OpenAI
+from pontoon.base.models import Locale
+
+
+class OpenAIService:
+    def __init__(self):
+        self.client = OpenAI()
+
+    def get_translation(
+        self, english_text, translated_text, characteristic, target_language_name
+    ):
+        try:
+            target_language = Locale.objects.get(name=target_language_name)
+        except Locale.DoesNotExist:
+            raise ValueError(
+                f"The target language '{target_language_name}' is not supported."
+            )
+
+        informal = (
+            f"You will be provided with text in English, along with its machine-generated translation in {target_language}. "
+            "Your objective is to revise the {target_language} translation to ensure it utilizes simpler language. Adhere to the following guidelines to achieve this:\n"
+            "- Clarity is Key: Ensure the translation conveys the original message in the clearest possible manner, without ambiguity or unnecessary complexity.\n"
+            "- Consistent Simplicity: Maintain a consistent level of simplicity throughout the translation.\n"
+            "The goal is to produce a translation that accurately reflects the original English text, but in a way that is more approachable and easier to understand for all {target_language} speakers."
+        )
+
+        formal = (
+            f"You will be provided with text in English, along with its machine-generated translation in {target_language}. "
+            "Your objective is to revise the {target_language} translation to ensure it adheres to a higher level of formality. Adhere to the following guidelines to achieve this:\n"
+            "- Adjust the Tone: Ensure the tone is respectful, polished, and devoid of colloquialisms or informal expressions commonly used in casual conversation.\n"
+            "- Formal Addressing: Where applicable, use formal modes of address.\n"
+            "- Consistency: Maintain a consistent level of formality throughout the translation, avoiding shifts in tone or style.\n"
+            "Your goal is to produce a translation that not only accurately conveys the meaning of the English text but also meets the expectations for formality in {target_language}-speaking professional or formal settings."
+        )
+
+        alternative = (
+            f"You will provided text in English along with its machine-generated translation in {target_language}. "
```

Comment:
"You will be provided with text in English, along with its machine-generated translation in {target_language}. "